### PR TITLE
add testing dev environment

### PR DESCRIPTION
**Pull request by mpodaniev**

### CODE REVIEW

**Comment by mpodaniev** [comment]
@dyaroman need review

File: readline.js [hunk 1]
```diff
@@ -1,28 +1,45 @@
...
 });
 
-const autoComplete = new AutoComplete({
-    name: 'projectName',
-    message: 'Choose your project',
-    limit: 10,
-    initial: 2,
-    choices: fs.readdirSync(projectsDirectoryPath)
+const env = new Select({
+  name: 'workflow',
```
Comment (dyaroman) [inline readline.js]:
```suggestion
  name: 'env',
```

File: utils.js [hunk 2]
```diff
@@ -64,15 +64,44 @@ const getScenariosForProject = function (getProjectPath, projectName) {
     return scenarios;
 }
 
+const getScenariosForHostProject = function (getProjectPath, projectName) {
+
+    const pages = settings.pages;
+
+    let scenarios;
```
Comment (dyaroman) [inline utils.js]:
```suggestion
```

File: utils.js [hunk 3]
```diff
@@ -64,15 +64,44 @@ const getScenariosForProject = function (getProjectPath, projectName) {
...
+
+    scenarios = pages.map(page => {
```
Comment (dyaroman) [inline utils.js]:
```suggestion
    const scenarios = pages.map(page => {
```

**Review by dyaroman — CHANGES_REQUESTED** [review]

**Review by dyaroman — APPROVED** [review]

**Pull request merged**